workshops.qmd

```diff
@@ -58,9 +58,10 @@ From the meeting page, copy the registration link to be included in your worksho
 
 ### Advertise
 
+-   Use [this spreadsheet](https://docs.google.com/spreadsheets/d/155tcna1dMWwsZymdKFZPCrnZf3xocw4hPv1JKwOu9MQ/edit#gid=0) to keep track of where you have advertised. 
```

Comment:
Nice, I like this idea. Does this supersede the (probably outdated) list here: https://cct-datascience.github.io/group-procedures/outreach-communication.html#where-to-announce-events ?